.github/workflows/ci.yml

```diff
@@ -45,13 +48,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
```

Comment:
This needs run_install false, so that cache can be set up next, and then pnpm i used -- otherwise pnpw install runs twice

This action handles it als for you: https://github.com/wyvox/action-setup-pnpm/blob/main/action.yml#L105